public/react/SaveReports/components/SavedReportDropdown/ReportControls.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { type FC } from "react";
+
+import DropdownMenu, { DropdownItem } from "@atlaskit/dropdown-menu";
+import LinkButton from "../../../components/LinkButton";
+import ChevronDown from "@atlaskit/icon/glyph/chevron-down";
+
+const defaultPrimaryButtonClasses = "ps-4 pe-2 py-2 rounded-md";
+
+interface ReportControlProps {
+  hasSelectedReport?: boolean;
+  isDirty?: boolean;
+  updateSelectedReport?: () => void;
+  openModal?: () => void;
+  resetChanges?: () => void;
+}
+
+const ReportControls: FC<ReportControlProps> = ({
+  hasSelectedReport = false,
+  isDirty = false,
+  updateSelectedReport = () => {},
```

Comment:
optional functions don't need to be defaulted

Reply:
if they dont have a default then you have to do an additional check before calling them